### PR TITLE
Look for multiple hosts holding the same VIP

### PR DESCRIPTION
**Pull request by omertuc**
This indicates keepalived issues / split brain, which might lead to
hosts trying to reach the API in servers aren't even running the
kube-apiserer

Example: https://issues.redhat.com/browse/AITRIAGE-8553?focusedId=23179977&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23179977

### CODE REVIEW

File: tools/triage/add_triage_signature.py [hunk 1]
```diff
@@ -2983,6 +2983,68 @@ def _process_ticket(self, url, issue_key):
...
+
+    def _process_ticket(self, url, issue_key):
+        cluster = get_metadata_json(url)["cluster"]
+
+        # SNO doesn't need balancing
+        if cluster.get("high_availability_mode") == "None":
+            return
+
+        # VIPs are not relevant with load balancer
+        if cluster.get("user_managed_networking") == True:
```
Comment (rccrdpccl) [inline tools/triage/add_triage_signature.py:3007]:
```suggestion
        if cluster.get("user_managed_networking"):
```

Comment (omertuc) [reply tools/triage/add_triage_signature.py:3007]: If `user_managed_networking` has any value other than `True` (e.g., `5`) then I prefer this `if` to not have its body executed, so your diff suggestion isn't technically equivalent.

The concept of values having "truthiness" is in my opinion a very unfortunate unnecessary complication that sadly most programming languages have which I prefer to not rely on.

Comment (rccrdpccl) [reply tools/triage/add_triage_signature.py:3007]: I agree, but linter disagrees :)

Comment (rccrdpccl) [reply tools/triage/add_triage_signature.py:3007]: We could ignore linting though

**Pull request merged**